### PR TITLE
Kwxm/minor eutxo spec updates

### PR DESCRIPTION
**Pull request by kwxm**
This makes a few small changes to the EUTXO document in the wake of the WTSC paper.  The definition of `Interval` now says that intervals can be unbounded, and that lets us just talk about `Interval[A]` everywhere.  This is in the final commit, so we can easily undo it if necessary.

### CODE REVIEW

**Comment by kwxm** [comment]
As it says near the top, we may want to replace `PendingTx` with `TxInfo`, but maybe we should change the implementation at the same time.

File: extended-utxo-spec/extended-utxo-specification.tex [hunk 1]
```diff
@@ -627,8 +633,6 @@ \subsection{Validity of EUTXO-1 transactions}
   \multicolumn{3}{l}{\lookupTx{\_}{\_} : \s{Ledger} \times \TxId \rightarrow \eutxotx{}}\\
```
Comment (michaelpj) [inline extended-utxo-spec/extended-utxo-specification.tex]: You can remove the underscores now and define it like a normal function.

File: extended-utxo-spec/extended-utxo-specification.tex [hunk 2]
```diff
@@ -135,7 +143,7 @@
 \newcommand{\outputref}{\mi{outputRef}}
 \newcommand{\txin}{\mi{in}}
 \newcommand{\id}{\mi{id}}
-\newcommand{\lookupTx}[2]{#1\langle#2\rangle}
+\newcommand{\lookupTx}[2]{\msf{lookupTx(#1, #2)}}
```
Comment (michaelpj) [inline extended-utxo-spec/extended-utxo-specification.tex]: Better to just make it a function name like all the others and drop the arguments.

File: extended-utxo-spec/extended-utxo-specification.tex [hunk 3]
```diff
@@ -1,3 +1,10 @@
+%% TODO
+%%
+%%   * Rename PendingTx to TxInfo, but not until the implementation also does so.
```
[the remark this answers is not in the document]
Comment (kwxm) [reply extended-utxo-spec/extended-utxo-specification.tex]: Yes!

File: extended-utxo-spec/extended-utxo-specification.tex (hunk 1 above)
Comment (kwxm) [reply extended-utxo-spec/extended-utxo-specification.tex]: Well spotted.

File: extended-utxo-spec/extended-utxo-specification.tex (hunk 2 above)
Comment (kwxm) [reply extended-utxo-spec/extended-utxo-specification.tex]: Done.

**Pull request merged**